src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/Rules/RazorComponentWithTargetPath.cs

```diff
@@ -203,7 +203,7 @@ private void InitializeFallbackRule()
                     Microsoft.Build.Framework.XamlTypes.IProjectSchemaNode root = ((Microsoft.Build.Framework.XamlTypes.IProjectSchemaNode)(System.Xaml.XamlServices.Load(xamlStream)));
                     System.Collections.Generic.IEnumerator<System.Object> ruleEnumerator = root.GetSchemaObjects(typeof(Microsoft.Build.Framework.XamlTypes.Rule)).GetEnumerator();
                     for (
-                    ; ((unboundRule == null) 
+                    ; ((unboundRule is null) 
                                 && ruleEnumerator.MoveNext()); 
```

Comment:
This is the most bizarre line wrapping I've seen this week :)

Reply:
Agreed. I'm gonna leave it as-is because of Chesterton’s fence but if someone who DOES understand what happened here wants to change it I support them.